### ISSUE TITLE
Maven package created JAR generates directory node_modules

### ISSUE THREAD

**Issue by lanmaster** [mvysny/vaadin14-embedded-jetty#1]
How to prevent creating that directory?
Fat Jar must include all application data except directory with Java libs and config files.
Is it downloading node files from intenet? Then it cannot be start without internet?

**Comment by mvysny** [comment]
Hi, 

TL;DR answer: create WAR with `mvn clean package -Pproduction`

Vaadin 14 NPM in development mode will download all npm packages when the server is started. So yes, in development mode the server can not be started without a working internet connection.

However, if you enable the production mode via `-Pproduction`, then WebPack should package all npm packages into WAR, and the server should then start even without working internet connection. If it doesn't, then it's a bug.

Please find more information on NPM and WebPack at https://vaadin.com/docs/v14/flow/v14-migration/v14-migration-guide.html

**Comment by lanmaster** [comment]
Cannot start WAR with 'java -jar 'command. JAR needed.

**Comment by lanmaster** [comment]
Oh sorry, my mistake. I see now that is WAR project you created.

**Comment by mvysny** [comment]
Ah right, now I've realized that:

1. The production mode is not enabled;
2. The war shouldn't be there since it's really a jar project

Let me look at this.
